.github/workflows/nightly.yml

```diff
@@ -207,7 +207,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Slack Workflow Notification
-        uses: Gamesight/slack-workflow-status@master
+        uses: Gamesight/slack-workflow-status@main
```

Comment:
Looks like these folks are still using `master` - don't think this can be changed

Reply:
fixed